src/Console/CreateInfrastructureCommand.cs

```diff
@@ -19,8 +19,7 @@ public CreateInfrastructureCommand(string name, string description = null) : bas
 
             Add(new Option<string>(new[] { "--site", "-s", }, Resources.INFRASTRUCTURE_SITE_HELP));
             Add(new Option<string>(new[] { "--customer", "-c", }, Resources.INFRASTRUCTURE_CUSTOMER_HELP));
-            Add(new Option<bool>(new[] { "--force", "-f", }, Resources.INFRASTRUCTURE_FORCE_HELP));
-            Add(new Option<int>(new[] { "--timeout", "-to", }, Resources.INFRASTRUCTURE_CREATION_SECONDS_TIMEOUT_HELP));
+            Add(new Option<bool>(new[] { "--ignoreIfExists", "-ie", }, Resources.INFRASTRUCTURE_IGNORE_IF_EXISTS_HELP));
```

Comment:
not sure if makes sense to include the alias for that option, -ie is weird and, in the context of the MES, ie means integration entry which may confuse the user

Reply:
what do you suggest? Previously I think -i but I can't because -i exists in CommonParameterExtension on interactive command. What you think literally '-ignore'?

Reply:
I'd say no alias at all

Reply:
-ignore isn't verbose enough, "what I'm why ignoring?". Since ignoreIfExists is the shorter version, I'd say it's best to remove the alias